lib/sasl.js

```diff
@@ -71,13 +71,13 @@ var PlainClient = function(username, password) {
     this.password = password;
 };
 
-PlainClient.prototype.start = function() {
+PlainClient.prototype.start = function(callback) {
```

Comment:
@pierreca - Can you also update the type definitions? In this way things are consistent.
[PlainClient.start](https://github.com/amqp/rhea/blob/master/typings/sasl.d.ts#L55) should now take a callback.